menas/ui/components/app.controller.js

```diff
@@ -33,6 +33,13 @@ sap.ui.define([
 
       this._app = this.byId("menasApp");
 
+      // register TNT icons
+      sap.ui.core.IconPool.registerFont({
```

Comment:
What's this for?

Reply:
This registers the namespace for technical (TNT) icons.

https://openui5.hana.ondemand.com/test-resources/sap/m/demokit/iconExplorer/webapp/index.html#/overview/SAP-icons-TNT